src/main/java/com/raisetech/mysql/sample/Controller/NameResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.raisetech.mysql.sample.Controller;
+
+import com.raisetech.mysql.sample.entity.Name;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class NameResponse {
+    private String name;
+
+
+    public NameResponse(Name name) {
+        this.name = name.getName();
+        
```

Comment:
Save Actionsで自動フォーマットする設定は入れていますか？
```suggestion
        this.name = name.getName();
```

Reply:
Save Actions設定しています。むらがありすみません、よく確認します。